endpointmanager/main.go

```diff
@@ -4,27 +4,66 @@ import (
 	"database/sql"
 	"fmt"
 
-	_ "github.com/lib/pq"
-)
+	"github.com/spf13/viper"
 
-// TODO: configuration file or commandline arguments
-const (
-	host     = "localhost"
-	port     = 5432
-	user     = "postgres"
-	password = "" // NOTE: this needs to be replaced with the appropriate password
-	dbname   = "postgres"
-	sslmode  = "disable"
+	_ "github.com/lib/pq"
 )
 
 var db *sql.DB
 
+func setupConfig() {
+	var err error
+
+	viper.SetEnvPrefix("lantern_endptmgr")
+	viper.AutomaticEnv()
+
+	err = viper.BindEnv("dbhost")
+	if err != nil {
```

Comment:
Suggest putting error checking and panic in it's own function and just calling the function between bindEnv calls